docker-compose.nessie.yaml

```diff
@@ -0,0 +1,23 @@
+version: '3'
```

Comment:
I wanted to avoid having a separate `docker-compose` file for nessie, because:
1. It Checks the file that we encourage our users to use in the quickstart guide - an important coverage IMO.
2. Having to maintain 2 separate docker-compose files is somewhat redundant and repeats 90% of the values.